docs/content/en/release-notes/_index.md

```diff
@@ -57,9 +57,168 @@ The K8ssandra helm chart deploys the following components. Some are optional, an
+k8ssandra       default         1               2021-08-17 18:58:03.831212 -0400 EDT    deployed        k8ssandra-1.3.0 
+```
+
+After about 10 minutes, verify the Cassandra Operator status:
+
+```bash
+kubectl describe cassdc dc1 | grep "Cassandra Operator Progress:"
+```
+**Output**:
+```
+  Cassandra Operator Progress:  Ready
+```
+
+Verify the server version:
+
+```bash
+kubectl describe cassdc dc1 | grep "Server Version:" 
+```
+**Output**:
+```
+  Server Version:  3.11.10
+```
+
+Verify the `num_tokens`, in this case defined by default value:
+
+```bash
+kubectl describe cassdc dc1 | grep "num_tokens:"
+```
+**Output**:
+```
+      num_tokens:                         256
+```
+
+#### Upgrades
+
+Upgrade to Cassandra 4.0.0 and explicitly sets the `num_tokens` to match the previous default (256).
+
+```bash
+helm upgrade k8ssandra -f k8ssandra-upgrade.values.yml k8ssandra/k8ssandra
+```
+
+k8ssandra-upgrade.values.yml:
+
+```yaml
+cassandra:
+  version: "4.0.0"
+  datacenters:
+    - 
+      name: dc1
+      size: 1
+      num_tokens: 256
+stargate:
+  enabled: false
+```
+
+(Again, `stargate` is disabled here just to expedite verification of the deployment. You could subsequently enable `stargate` by editing the values YAML and submitting a follow-up `helm upgrade` command.)
+
+**Resulting deployment:**
+
+After about 10 minutes, verify the Cassandra Operator status:
+
+```bash
+kubectl describe cassdc dc1 | grep "Cassandra Operator Progress:"                              
+```
+**Output**:
+```
+  Cassandra Operator Progress:  Ready
+```
+
+Check the pods:
+
+```bash
+kubectl get pods
+```
+**Output**:
+```
+NAME                                                READY   STATUS      RESTARTS   AGE
+k8ssandra-cass-operator-5bbf6584b9-pgldj            1/1     Running     0          9m23s
+k8ssandra-crd-upgrader-job-k8ssandra-skrjh          0/1     Completed   0          2m6s
+k8ssandra-dc1-default-sts-0                         2/2     Running     0          106s
+k8ssandra-grafana-679b4bbd74-ngrz2                  2/2     Running     0          9m23s
+k8ssandra-kube-prometheus-operator-85695ffb-qhqc7   1/1     Running     0          9m23s
+k8ssandra-reaper-694cf49b96-jgnvg                   0/1     Running     1          6m12s
+k8ssandra-reaper-operator-5c6cc55869-tdhjw          1/1     Running     0          9m23s
+prometheus-k8ssandra-kube-prometheus-prometheus-0   2/2     Running     1          9m14s
+```
+
+Verify the server version:
+
+```bash
+kubectl describe cassdc dc1 | grep "Server Version:" 
+```
+**Output**:
+```
+  Server Version:  4.0.0
+```
+
+Verify `num_tokens`:
+
+```bash
+kubectl describe cassdc dc1 | grep "num_tokens:"
+```
+**Output**:
+```
+      num_tokens:                         256
+```
+
+Because the explicitly set `num_tokens` value in 4.0 matches the previous `num_tokens` default in 3.11 (256), Cassandra Operator starts. 
```

Comment:
It may be more correct here to say that `Cassandra` starts, rather than the `Operator`

Reply:
Good - thx!